.github/workflows/pr.yml

```diff
@@ -0,0 +1,30 @@
+name: pr.yml
+
+on:
+  pull_request: {}
+
+jobs:
+  actionlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Download actionlint
+        id: get_actionlint
+        run: bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash)
```

Comment:
```suggestion
        run: bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/be0549f3c56c3bc347c94e43622466084fafc61d/scripts/download-actionlint.bash)
```

Reply:
I prefer pointing to specific commits to download bash scripts, if we point to a branch only the script can be replaced without our knowledge :)

Reply:
```suggestion
        run: bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/03d0035246f3e81f36aed592ffb4bebf33a03106/scripts/download-actionlint.bash)
```

Reply:
Yep, that's a really good point. This is how they document setting up in GitHub Actions but that doesn't mean it's a good idea. 

I'm suggesting the hash of their last release (1.7.7) instead of the latest change on `main`.

Reply:
Updated in commit https://github.com/holochain/actions/pull/1/commits/36f91276cef19886e1b1509815f69653f9bc6594